graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -173,7 +173,9 @@ def _get_fields(ast):
             # If we ever allow repeated field names,
             # then we have to change the Location naming scheme to reflect the repetitions
             # and disambiguate between Recurse and Traverse visits to a Location.
-            raise GraphQLCompilationError(u'Encountered repeated field name: {}'.format(name))
+            raise GraphQLCompilationError(u'Encountered repeated field name: {}. To use '
+                                          u'multiple directives on a field, write the other '
+                                          u'directives directly below the first one.'.format(name))
```

Comment:
I think this is solving a real problem that is worth solving. We just have to be mindful that our solution doesn't cause more damage than good.

There are other situations where this error can happen, it's not just multiple directives. In principle, the following will raise this error as well:
```
{
    Animal {
        name
        name
        alias @output(out_name: "alias")
    }
}
```

If I were to write the above, and then see this error message in response, that would be very confusing.

Also, "directly below" is misleading and unnecessarily strict -- not all editors have "directly below" as an option (e.g. consider the default Flask-JSONRPC frontend). How about "adjacent to" instead?